packages/contracts/src/RelayHub.sol

```diff
@@ -192,11 +192,29 @@ contract RelayHub is IRelayHub, Ownable, ERC165 {
     function withdraw(uint256 amount, address payable dest) public override {
         address payable account = payable(msg.sender);
         require(balances[account] >= amount, "insufficient funds");
+        uint256[] memory amounts = new uint256[](1);
+        address payable[] memory destinations = new address payable[](1);
+        amounts[0] = amount;
+        destinations[0] = dest;
+        withdrawMultiple(amounts, destinations);
```

Comment:
I understand your argument regarding withdraw[single]. you're right it is better to leave it as a helper.

Reply:
Thank you. If you know how to create dynamically-sized in-memory array without declaring it please tell me. I could not explicitly convert ```uint[1]``` to ``` uint[]```.